docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -875,6 +887,89 @@ To process the received search command on the server side, follow these steps:
+   ```
+   ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_PLAY, MC_ABILITY_SUPPORTED_YES);
+   ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_PAUSE, MC_ABILITY_SUPPORTED_NO);
+   ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_STOP, MC_ABILITY_SUPPORTED_YES);
+   ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_NEXT, MC_ABILITY_SUPPORTED_NO);
+   ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_PREV, MC_ABILITY_SUPPORTED_YES);
+   ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_FAST_FORWARD, MC_ABILITY_SUPPORTED_NO);
+   ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_REWIND, MC_ABILITY_SUPPORTED_YES);
+   ret = mc_server_set_playback_ability(g_mc_server, MC_PLAYBACK_ACTION_TOGGLE_PLAY_PAUSE, MC_ABILITY_SUPPORTED_NO);
+   
+   ret = mc_server_update_playback_ability(g_mc_server);
+   ```
+   
+   To update the display mode ability, set the ability using 'mc_server_set_display_mode_ability()':
+   ```
+   ret = mc_server_set_display_mode_ability(g_mc_server, MC_DISPLAY_MODE_LETTER_BOX | MC_DISPLAY_MODE_ORIGIN_SIZE | MC_DISPLAY_MODE_FULL_SCREEN | MC_DISPLAY_MODE_CROPPED_FULL, MC_ABILITY_SUPPORTED_YES);
+   ```
+   
+   To update the display rotation ability, set the ability using 'mc_server_set_display_rotation_ability()':
+    ```
+   ret = mc_server_set_display_rotation_ability(g_mc_server, MC_DISPLAY_ROTATION_NONE | MC_DISPLAY_ROTATION_90 | MC_DISPLAY_ROTATION_180 | MC_DISPLAY_ROTATION_270, MC_ABILITY_SUPPORTED_YES);
+   ```
+   
+   In case of other abilities, set the ability using 'mc_server_set_ability_support()', to update an ability:
+   For example, to update shuffle and repeat ability, set the ability using the 'mc_server_set_ability_support()':
+   ```
+   ret = mc_server_set_ability_support(g_mc_server, MC_ABILITY_SHUFFLE, MC_ABILITY_SUPPORTED_YES);
+   ret = mc_server_set_ability_support(g_mc_server, MC_ABILITY_REPEAT, MC_ABILITY_SUPPORTED_NO);
+   ```
+
+To retrieve the abilities on the client side, follow these steps:
+
+1. Create the media controller client handle using the `mc_client_create()`:
+
+   ```
+   ret = mc_client_create(&g_client_h);
+   ```
+
+2. Define the callback that gets invoked when the client receives the change in abilities.
+   To define a callback for a playback ability:
+   ```
+   ret = mc_client_set_playback_ability_updated_cb(g_client_h, _mc_playback_ability_updated_cb, NULL);
+   ```
+   
+   - `mc_client_set_playback_ability_updated_cb()`: For playback ability change.
+   - `mc_client_set_display_mode_ability_updated_cb()`: For display mode ability change.
+   - `mc_client_set_display_rotation_ability_updated_cb()`: For display rotation ability change.
+   - `mc_client_set_ability_support_updated_cb()`: For other abilities change.
+   
+3. Or, client can get the server's ability directly.
```

Comment:
The client can get the server ability directly.
The following example demonstrates setting up the server abilities:

Reply:
I think the suggestion is too wordy. 

The client can get the server ability directly. For example to get server playback ability:

Would be enough.

Reply:
Done